docs/contributing/building/nocore-ebpf.md

```diff
@@ -118,36 +118,38 @@ If you install the non CO-RE eBPF object and run **tracee-ebpf** in an
+{"level":"debug","ts":1670617146.8965306,"msg":"BTF","pkg":"initialize","file":"bpfobject.go","line":40,"bpfenv":false,"btfenv":false,"vmlinux":true}
+{"level":"debug","ts":1670617146.8968363,"msg":"BPF: using embedded BPF object","pkg":"initialize","file":"bpfobject.go","line":69}
+{"level":"debug","ts":1670617147.0876234,"msg":"unpacked CO:RE bpf object file into memory","pkg":"initialize","file":"bpfobject.go","line":144}
+
 ```
 
 One way of forcing **tracee-ebpf** to use non CO-RE eBPF object, even in a kernel
 that supports CO-RE, is by setting the `TRACEE_BPF_FILE` environment, like this:
 
 ```
-$ sudo TRACEE_BPF_FILE=/tmp/tracee/tracee.bpf.5_4_0-91-generic.v0_6_5-80-ge723a22.o ./dist/tracee-ebpf --debug -o option:parse-arguments --trace comm=bash --trace follow
-OSInfo: PRETTY_NAME: "Ubuntu 20.04.3 LTS"
-OSInfo: VERSION_ID: "20.04"
-OSInfo: VERSION_CODENAME: focal
-OSInfo: KERNEL_RELEASE: 5.4.0-91-generic
-OSInfo: ARCH: x86_64
-OSInfo: VERSION: "20.04.3 LTS (Focal Fossa)"
-OSInfo: ID: ubuntu
-OSInfo: ID_LIKE: debian
-BTF: bpfenv = true, btfenv = false, vmlinux = false
-BPF: using BPF object from environment: /tmp/tracee/tracee.bpf.5_4_0-91-generic.v0_6_5-80-ge723a22.o
+$ sudo TRACEE_BPF_FILE=/tmp/tracee/tracee.bpf.5_4_0-132-generic.v0_8_0-rc-2-342-g29fe4c74.o ./dist/tracee-ebpf --log debug -o option:parse-arguments --trace comm=bash --trace follow
+{"level":"debug","ts":1670617459.8095431,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"PRETTY_NAME","OS_KERNEL_RELEASE":"\"Ubuntu 20.04.5 LTS\""}
+{"level":"debug","ts":1670617459.810961,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"VERSION_ID","OS_KERNEL_RELEASE":"\"20.04\""}
+{"level":"debug","ts":1670617459.8109868,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"VERSION_CODENAME","OS_KERNEL_RELEASE":"focal"}
+{"level":"debug","ts":1670617459.811008,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"KERNEL_RELEASE","OS_KERNEL_RELEASE":"5.4.0-132-generic"}
+{"level":"debug","ts":1670617459.81103,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"ARCH","OS_KERNEL_RELEASE":"x86_64"}
+{"level":"debug","ts":1670617459.8110468,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"VERSION","OS_KERNEL_RELEASE":"\"20.04.5 LTS (Focal Fossa)\""}
+{"level":"debug","ts":1670617459.8110704,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"ID","OS_KERNEL_RELEASE":"ubuntu"}
+{"level":"debug","ts":1670617459.81109,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"ID_LIKE","OS_KERNEL_RELEASE":"debian"}
+{"level":"debug","ts":1670617459.8134959,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":101,"security_lockdown":"none"}
+{"level":"debug","ts":1670617459.818108,"msg":"BTF","pkg":"initialize","file":"bpfobject.go","line":40,"bpfenv":true,"btfenv":false,"vmlinux":true}
```

Comment:
same here - environments don't correspond (and debug messages are different)

Reply:
ditto. tks.